write_header.py

```diff
@@ -0,0 +1,40 @@
+#! /usr/bin/python3
+import argparse
+from argparse import RawTextHelpFormatter
+
+description="Adds the first part of a VCF header up until the end of the ##INFOs, \nthen adds the newly generated ##configs and ##reference, and finally add the rest of the header."
```

Comment:
A header can be sorted any way, which means there could be useful information after an INFO entry.

Reply:
I wasn't sure about the order of VCF headers, so if they're un-ordered does this mean I can just delete lines starting with ##contig from the old header, regenerate them using the new contig names, and paste them after the old header?

Reply:
Meta-information (`##`) lines can in theory go in any order, except for the very first `##fileformat` line. In the VCF specification, sections describing different meta-information lines are arranged in some order, but it never says that lines themselves must be arranged: https://samtools.github.io/hts-specs/VCFv4.2.pdf

I suggest you use `bcftools reheader` for replacing contig names

Reply:
I've changed how I generate the header, it now just simply adds everything that isn't a contig to my newly generated contigs list to make the new header. If there is still a problem with this, let me know and I will use `bcftools reheader`.